packages/core/test/integration/query-interface.test.js

```diff
@@ -480,329 +480,4 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
       });
     }
   });
-
-  describe('describeForeignKeys', () => {
-    beforeEach(async function () {
-      await this.queryInterface.createTable('users', {
-        id: {
-          type: DataTypes.INTEGER,
-          primaryKey: true,
-          autoIncrement: true,
-        },
-      });
-      await this.queryInterface.createTable('hosts', {
-        id: {
-          type: DataTypes.INTEGER,
-          primaryKey: true,
-          autoIncrement: true,
-        },
-        admin: {
-          type: DataTypes.INTEGER,
-          references: {
-            table: 'users',
-            key: 'id',
-          },
-        },
-        operator: {
-          type: DataTypes.INTEGER,
-          references: {
-            table: 'users',
-            key: 'id',
-          },
-          onUpdate: 'cascade',
-        },
-        owner: {
-          type: DataTypes.INTEGER,
-          references: {
-            table: 'users',
-            key: 'id',
-          },
-          onUpdate: 'cascade',
-          onDelete: 'set null',
-        },
-      });
-    });
-
-    it('should get a list of foreign keys for the table', async function () {
```

Comment:
I'm likely missing it, but is this test coming back in any form?

Reply:
This is in the `add-show-remove-constraint.test.ts` file where for each constraint type, there is a test to add, show and list the constraints for a table.

Reply:
Fixed in previous PR